helm/values-dev.yaml

```diff
@@ -85,8 +88,13 @@ microservice-chart:
     NPG_URI: "https://localhost/npg"
     NPG_READ_TIMEOUT: 10000
     NPG_CONNECTION_TIMEOUT: 10000
+    SESSION_URL_BASEPATH: "https://dev.checkout.pagopa.it"
```

Comment:
Why don't we call this variable **NPG_URL_BASEPATH** that way we can also use it for the call to retrieve the bin

Reply:
Is mapped through `SessionUrlConfig` and used in [PaymentMethodsService](https://github.com/pagopa/pagopa-ecommerce-payment-methods-service/pull/85/files#diff-39058d915edfb89e8569cc2b446a7fa1277bd8a4c14f324bef41e8deca210736R267-R272)